mlir/lib/ExecutionEngine/ROCm/rocm-runtime-wrappers.cpp

```diff
@@ -381,12 +381,18 @@ extern "C" void mgpuMemCopy2DFloat(float *sourceAllocated, float *sourceAligned,
             static_cast<hipMemcpyKind>(copyDirection));
 }
 
-short randomValue(short min, short max) {
+short random_int_value(short min, short max) {
```

Comment:
@yiqian1 please help up come with new flags to switch between int or float RNG, as well as the range & error thresholds.